spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'Doe John',
+      photo: 'http://placeimg.com/640/480/any',
+      bio: 'I am a develpoer',
+      post_counter: 0
+    )
+
+    @post1 = Post.create(
+      author: @user,
+      title: 'My test post',
+      text: 'This is my test post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+
+    @post2 = Post.create(
+      author: @user,
+      title: 'My second test post',
+      text: 'This is my second post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
```

Comment:
- [ ] Please add a test that asserts you can see a post's title.

Reply:
- [ ] Please add a test that asserts you can see the first comments on a post.

Reply:
Please add a test that redirects to the post's show page when you click on a post.